site/src/pages/UserSettingsPage/WorkspaceProxyPage/hooks.ts

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from "@tanstack/react-query"
+import { getWorkspaceProxies } from "api/api"
+
+// Loads all workspace proxies
```

Comment:
I don't think we need this secondary hook. Instead, we should roll whatever you need out of this into your `ProxyContext`. We already make a call in there to `getWorkspaceProxies` .

Reply:
You are right.